soc/riscv/esp32c3/Kconfig.soc

```diff
@@ -11,6 +11,9 @@ config SOC_ESP32C3
 	select PINCTRL
 	select XIP
 	select HAS_ESPRESSIF_HAL
+	select RISCV_ISA_RV32I
+	select RISCV_ISA_EXT_M
+	select RISCV_ISA_EXT_A
```

Comment:
I may be confusing things, but https://www.espressif.com/sites/default/files/documentation/esp32-c3_datasheet_en.pdf mentions that the core is RV32IMC - shouldn't then be `RISCV_ISA_EXT_C` instead of `RISCV_ISA_EXT_A` here?